_posts/2016-05-20-release-2-7-0.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Release 2.7.0"
+date: 2016-05-20
+author: "@atoum/RMs"
+categories: release
+---
+
+## Stats
+
+* 13 commits,
+* 82 files changed,
+* 4 contributors,
+* 1 new features,
+* 2 bug fixes.
+
+## What's new
+
+### Use telemetry by atoum itself
+
+As we recently announced, we now provide a collaborative platform to share tests results and statistics called [Telemetry].
+
+According to the atoum's philosophy, atoum itself tests execution now send results to the telemetry platform.
+
+## Bugfixes
+
+### Reports override correctly when using -ulr/-utr
+
+Using `--use-light-report` or `--use-tap-report` no longer remove all defined reports (like Telemetry) but only those who are in conflict with light or TAP reports.
+
+### Assertions on PHP 7 exceptions/throwables/errors are now working correctly
```

Comment:
I would have written: “Assertions are now working expectedly on PHP7”. No need to list them in the title.